TS4114 error on new generated commands

**Is this a bug with the oclif CLI? Or the oclif framework?**
oclif CLI

**Describe the bug**
When generating a new command, I get a TS4114 error telling me that the static properties in the new command class need to indicate that they are an override. I was going to put through a PR for [the template](https://github.com/oclif/oclif/blob/main/templates/src/command.ts.ejs) but figured it would be best to start a discussion here first. 

**To Reproduce**
Steps to reproduce the behavior:

1. Install `@oclif/core` in a project
2. Run `npm exec oclif generate command hello`
3. Build TypeScript with `tsc`
4. See 4 errors:

```bash
src/commands/hello.ts:4:10 - error TS4114: This member must have an 'override' modifier because it overrides a member in the base class 'Command'.

4   static args = {
           ~~~~

src/commands/hello.ts:8:10 - error TS4114: This member must have an 'override' modifier because it overrides a member in the base class 'Command'.

8   static description = "describe the command here";
           ~~~~~~~~~~~

src/commands/hello.ts:10:10 - error TS4114: This member must have an 'override' modifier because it overrides a member in the base class 'Command'.

10   static examples = ["<%= config.bin %> <%= command.id %>"];
            ~~~~~~~~

src/commands/hello.ts:12:10 - error TS4114: This member must have an 'override' modifier because it overrides a member in the base class 'Command'.

12   static flags = {
```

**Expected behavior**
No TS compile errors

**Your Environment (please complete the following information):**

```bash
$ npm exec oclif -- --version
oclif/4.7.5 darwin-arm64 node-v18.19.1
```

```bash
$ sw_vers
ProductName:            macOS
ProductVersion:         14.3
BuildVersion:           23D56
```

```
$ zsh --version
zsh 5.9 (arm-apple-darwin23.0.0)
```

--- a/templates/src/command.ts.ejs
+++ b/templates/src/command.ts.ejs
@@ -1,17 +1,17 @@
 import {Args, Command, Flags} from '@oclif/core'
 
 export default class <%- className %> extends Command {
-  static args = {
+  static override args = {
     file: Args.string({description: 'file to read'}),
   }
 
-  static description = 'describe the command here'
+  static override description = 'describe the command here'
 
-  static examples = [
+  static override examples = [
     '<%%= config.bin %> <%%= command.id %>',
   ]
 
-  static flags = {
+  static override flags = {
     // flag with no value (-f, --force)
     force: Flags.boolean({char: 'f'}),
     // flag with a value (-n, --name=VALUE)

